newton/_src/utils/import_usd.py

```
@@ -2842,14 +2680,14 @@ def _resolve_contact_attr(key, _prim=prim):
             )
             # Collect engine-specific attributes for the articulation root on first encounter
             if collect_schema_attrs:
-                R.collect_prim_attrs(articulation_prim)
+                resolver_manager.collect_prim_attrs(articulation_prim)
                 # Also collect on the parent prim (e.g. Xform with PhysxArticulationAPI)
                 try:
                     parent_prim = articulation_prim.GetParent()
                 except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)